universal/src/crypto/helpers.cpp

```diff
@@ -186,9 +197,22 @@ std::unique_ptr<::BIO, decltype(&::BIO_free_all)> MakeBioMemoryBuffer() {
   return {::BIO_new(BIO_s_mem()), &::BIO_free_all};
 }
 
+#ifndef USERVER_FEATURE_WOLFSSL
 std::unique_ptr<::BIO, decltype(&::BIO_free_all)> MakeBioSecureMemoryBuffer() {
   return {::BIO_new(BIO_s_secmem()), &::BIO_free_all};
 }
+#endif
+
+#ifdef USERVER_FEATURE_WOLFSSL
+#ifndef EVP_PKEY_CTX_set_rsa_pss_saltlen
+#warning "EVP_PKEY_CTX_set_rsa_pss_saltlen() is undefined by wolfSSL"
+#define EVP_PKEY_CTX_set_rsa_pss_saltlen(...) 0
+#endif
+#ifndef EVP_PKEY_CTX_set_rsa_mgf1_md
+#warning "EVP_PKEY_CTX_set_rsa_mgf1_md() is undefined by wolfSSL"
```

Comment:
why is it possible?

Reply:
These functions are missed in the header [wolfssl/openssl/evp.h](https://github.com/wolfSSL/wolfssl/blob/master/wolfssl/openssl/evp.h). Sadly, buy I have no idea currently how to make workaround for these functions.